src/twisted/python/_dist3.py

```diff
@@ -0,0 +1,185 @@
+# -*- test-case-name: twisted.python.test.test_dist3 -*-
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+
+"""
+Python 3 porting metadata. Only necessary while parts of Twisted are unported.
+
+@var notPortedModules: A list of fully-qualified Python names that correspond
+    to modules in Twisted are not ported, for exclusion during installation.
+"""
+
+from __future__ import absolute_import, division
+
+notPortedModules = [
+    "twisted.conch.tap",
+    "twisted.conch.ttymodes",
+    "twisted.internet._ssl",
```

Comment:
Hooray :).  In that case it wouldn't be able to land if this wasn't fixed anyway; awesome.